checks.md

```diff
@@ -1,4 +1,5 @@
 - [ ] **Fit**: The package meets criteria for [fit](https://www.pyopensci.org/peer-review-guide/about-peer-review/aims-and-scope.html#package-scope) and [overlap](https://www.pyopensci.org/peer-review-guide/about-peer-review/aims-and-scope.html#package-overlap).
```

Comment:
i think this file duplicates editor-in-chief-checks. Maybe we delete and rename the checks file to 

pre-review-package-requirements.md or something that more clearly speaks to the content in the file vs who uses the content especially since the EIC may ask an editor to perform the checks.

Reply:
sweet. ok i'll rename! thanks!

Reply:
i'm actually confused as to where this lives because it duplicates content in another checks template. 😆 oh my 🤯

Reply:
ok i've edited this file too... i think it's more clear now.